jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_LETKF

```diff
@@ -16,6 +16,9 @@ YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx \
    COM_OCEAN_HISTORY_PREV:COM_OCEAN_HISTORY_TMPL \
    COM_ICE_HISTORY_PREV:COM_ICE_HISTORY_TMPL
 
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_OBS 
+ROTDIR=${COMROOT} RUN=enkf${RUN} YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx COM_TOP_PREV_ENS:COM_TOP_TMPL
```

Comment:
`COM_TOP` should only be used is _absolutely_ necessary for the top of the tree (which I think only cleanup should need). Here it is being used in this job to generate paths to data. Jobs should not be prescribing directory paths. Instead the specific COM templates defined in `config.com` should be used to generate the paths (e.g. `COM_OCEAN_HISTORY_TMPL`), iterating over the memdirs if necessary. Otherwise, if `COM_OCEAN_HISTORY_TMPL` were to change in `config.com`, this code would break.

Reply:
Verified that `COM_TOP` is not used anymore.